adafruit_icm20x.py

```diff
@@ -140,7 +140,7 @@ class ICM20X:  # pylint:disable=too-many-instance-attributes
 
 
     :param ~busio.I2C i2c_bus: The I2C bus the ICM20X is connected to.
-    :param address: The I2C slave address of the sensor
+    :param int address: The I2C slave address of the sensor
```

Comment:
Please update the use of the word `slave` here to `peripheral` to match the CircuitPython Design Guide terminology.

Reply:
Will do thanks :)

Reply:
I will go back and correct this and all the busio designations in previous PRs. Thanks @kattni

Reply:
@kattni I updated this according to your suggestion and will continue to use the design guide for the comments

Reply:
This is solved in https://github.com/adafruit/Adafruit_CircuitPython_ICM20X/pull/12/commits/5c27ce9680bb3ee3f51d3810496bea7ea2aab812